packages/cli/src/util/update-notifier/update-worker.js

```diff
@@ -57,11 +64,15 @@ async function fetchLatest({ cacheFile, distTag, name, updateCheckInterval }) {
     notified: false,
     version,
   });
+
+  // success, no more need for the timeout
+  clearTimeout(timer);
 }
 
-/**
- * Signal the parent process we're ready to fetch.
- */
+// signal the parent process we're ready
 if (process.connected && process.send) {
   process.send({ type: 'ready' });
+} else {
+  console.error('No IPC bridge detected, exiting');
```

Comment:
**suggestion (soft-blocking):** move to debug log

In this or a follow-up PR: This will be confusing for a user and not actionable. Can we move it to a debug log?

Reply:
Users will never see this because `process.connected` and `process.send` will always be defined in the context of the Vercel CLI.